Release `v0.1.1`

robbot006 opened the pull request:
This is a draft of changelog for the `v0.1.1` release.
Here you can find all related translations changes: [v0.1...master](https://github.com/flarum-lang/korean/compare/v0.1...master).

> ### ⚠️ Do not merge this pull request manually ⚠️
> 
> You should [approve](https://github.com/rob006-software/flarum-translations/wiki/How-to-approve-release-pull-request) this pull request instead. After approving, I will automatically update it (release date needs to be adjusted), merge, and tag a new release.

<details>
<summary>Show release notes preview</summary>

## v0.1.1

<!-- release-notes-begin -->
**General changes**:

* Updated Flarum core translations.
* Updated validation translations.


**Updated translations for extensions**:

* [`flarum/akismet`](https://github.com/flarum/akismet)
* [`flarum/approval`](https://github.com/flarum/approval)
* [`flarum/emoji`](https://github.com/flarum/emoji)
* [`flarum/flags`](https://github.com/flarum/flags)
* [`flarum/likes`](https://github.com/flarum/likes)
* [`flarum/lock`](https://github.com/flarum/lock)
* [`flarum/markdown`](https://github.com/flarum/markdown)
* [`flarum/mentions`](https://github.com/flarum/mentions)
* [`flarum/nicknames`](https://github.com/flarum/nicknames)
* [`flarum/pusher`](https://github.com/flarum/pusher)
* [`flarum/statistics`](https://github.com/flarum/statistics)
* [`flarum/sticky`](https://github.com/flarum/sticky)
* [`flarum/subscriptions`](https://github.com/flarum/subscriptions)
* [`flarum/suspend`](https://github.com/flarum/suspend)
* [`flarum/tags`](https://github.com/flarum/tags)
* [`fof/masquerade`](https://github.com/FriendsOfFlarum/masquerade)


All changes: [v0.1...v0.1.1](https://github.com/flarum-lang/korean/compare/v0.1...v0.1.1).



<!-- release-notes-end -->

</details>

robbot006 committed:
Update changelog.

rob006 commented:
@hahagu Just as a note, this is an automated PR generated by bot. The idea is to simplify release process for language pack maintainers - all you need is to approve this pull request and bot will handle the rest. You can read more about the process at forum: https://discuss.flarum.org/d/20807-simplify-translation-process-with-weblate/76 . If you don't want to use this feature, let me know.

robbot006 committed:
Update changelog.

robbot006 commented:
Success! Now you can announce new release on [forum](https://discuss.flarum.org/t/languages):

~~~markdown
## Version [`v0.1.1`](https://github.com/flarum-lang/korean/releases/tag/v0.1.1)

**General changes**:

* Updated Flarum core translations.
* Updated validation translations.


**Updated translations for extensions**:

* [`flarum/akismet`](https://github.com/flarum/akismet)
* [`flarum/approval`](https://github.com/flarum/approval)
* [`flarum/emoji`](https://github.com/flarum/emoji)
* [`flarum/flags`](https://github.com/flarum/flags)
* [`flarum/likes`](https://github.com/flarum/likes)
* [`flarum/lock`](https://github.com/flarum/lock)
* [`flarum/markdown`](https://github.com/flarum/markdown)
* [`flarum/mentions`](https://github.com/flarum/mentions)
* [`flarum/nicknames`](https://github.com/flarum/nicknames)
* [`flarum/pusher`](https://github.com/flarum/pusher)
* [`flarum/statistics`](https://github.com/flarum/statistics)
* [`flarum/sticky`](https://github.com/flarum/sticky)
* [`flarum/subscriptions`](https://github.com/flarum/subscriptions)
* [`flarum/suspend`](https://github.com/flarum/suspend)
* [`flarum/tags`](https://github.com/flarum/tags)
* [`fof/masquerade`](https://github.com/FriendsOfFlarum/masquerade)


All changes: [v0.1...v0.1.1](https://github.com/flarum-lang/korean/compare/v0.1...v0.1.1).

In order to update run these commands:

```console
composer update flarum-lang/korean
php flarum cache:clear
```

~~~
